frontend/src/e2e-test/dev-api/fixtures.ts

```diff
@@ -323,6 +323,8 @@ export const clubFixture: Daycare = {
   areaId: careAreaFixture.id,
   name: 'Alkuräjähdyksen kerho',
   type: ['CLUB'],
+  dailyPreschoolTime: null,
+  dailyPreparatoryTime: null,
```

Comment:
Not sure if the db constraint should be loosened to always allow non-null values so we wouldn't need this kind of boilerplate to override the defaults whenever overriding the type field?